cgi-bin/dashboard-aii.cgi

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use Config::Simple;
+use JSON::XS;
+use Switch;
+use CGI;
+
+# Configuration file to boot from HD. Following pxelinux convention
+# it must be called default so if the <IpAddressInHex> link
+# is missing the node will boot from the hard disk
+my $boothd = "default";
+
+# Quattor profiles
+my $web_root = "/var/www/html";
+my $profiles_path = "profiles";
```

Comment:
as this is an `aii-shellfe` box, you can use indeed the settings from `/etc/aii/aii-shellfe.conf` as you do below, and read the values from there, e.g. `cdburl`
we have `cdburl=file:///path/to/profiles` in our AII servers